packages/extension/package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "main": "index.js",
   "scripts": {
-    "build": "gulp build",
+    "build": "cross-env NODE_OPTIONS=\"$(node ../../scripts/use-legacy-openssl.js)\" gulp build",
```

Comment:
Does this work in Windows? IDK how the command substitution is handled cross-platform

Reply:
Good point I'll give it a test!

Reply:
Didn't work on Windows, changed to running webpack in a child process so the env variables could be manipulated more easily